pkg/kubelet/container/runtime_test.go

```diff
@@ -0,0 +1,786 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package container
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func TestParseContainerID(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    string
+		expected ContainerID
+	}{
+		{
+			name:     "valid docker container id",
+			input:    `"docker://abc123"`,
```

Comment:
nit: some input values are quoted while some are not - "docker://abc123" and `"docker://abc123"`

Reply:
ah my bad!